migrations/1595584859678-RemoveStudentDeletePermissionForTeachers.js

```diff
@@ -0,0 +1,53 @@
+const mongoose = require('mongoose');
+// eslint-disable-next-line no-unused-vars
+const { info, error } = require('../src/logger');
+
+const { connect, close } = require('../src/utils/database');
+
+const RoleModel = mongoose.model('role20202707', new mongoose.Schema({
+    name: { type: String, required: true },
+    permissions: [{ type: String }],
+}, {
+    timestamps: true,
```

Comment:
Codacy found an issue: [Expected indentation of 1 tab but found 4 spaces.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5914670)